doc/articles/get-started-licensing.md

```diff
@@ -0,0 +1,63 @@
+---
+uid: Uno.GetStarted.Licensing
+---
+
+# Sign in with Uno Platform
+
+Sign in with your Uno Platform account directly in your favorite IDE—Visual Studio, VS Code, or Rider—to unlock powerful tools like Hot Reload, helping you speed up development. With a single registration, you also get early access to new features and the opportunity to connect with the Uno community to share feedback and network.
+
+## Create your account
+
+1. Go to our website, [platform.uno](https://platform.uno/), and click on the **Sign in** button in the top right corner, or go directly to [platform.uno/my-account](https://platform.uno/my-account).
+2. Enter your email address and click on **Register**.
+3. On the registration page, fill in your information, including your email, name, country, and password. Once done, click on **Sign up**.
```

Comment:
```suggestion
3. On the registration page, fill in your information. Once done, click on **Sign up**.
```
Maybe best to not go to much into details of the form in case the fields of the form change with time no?